frontends/mit-open/src/page-components/TestimonialCard/TestimonialCard.tsx

```diff
@@ -0,0 +1,184 @@
+  margin: "0 auto",
+  padding: "0 22px",
+}))
+
+const QuoteLeader = styled.div(({ theme }) => ({
+  width: "100%",
+  height: "32px",
+  fontSize: "60px",
+  lineHeight: "60px",
+  color: theme.custom.colors.silverGray,
+}))
+
+const QuoteBody = styled.div(({ theme }) => ({
+  width: "100%",
+  display: "flex",
+  [theme.breakpoints.down("md")]: {
+    flexDirection: "column",
+  },
+}))
+
+const AttestationBlock = styled.div(({ theme }) => ({
+  width: "auto",
+  flexGrow: "5",
+  ...theme.typography.h5,
+  justifyContent: "top",
+}))
+
+const AttestantBlock = styled.div(({ theme }) => ({
+  width: "248px",
+  display: "flex",
+  [theme.breakpoints.down("md")]: {
+    marginTop: "24px",
+  },
+}))
+
+const AttestantAvatar = styled.div({
+  marginRight: "12px",
+  img: {
+    objectFit: "cover",
+    borderRadius: "40px",
+    width: "40px",
+    height: "40px",
+    boxShadow:
+      "0px 2px 4px 0px rgba(37, 38, 43, 0.10), 0px 2px 4px 0px rgba(37, 38, 43, 0.10)",
+  },
+})
+
+const AttestantNameBlock = styled.div(({ theme }) => ({
+  flexGrow: "1",
+  width: "auto",
+  color: theme.custom.colors.lightGray2,
+}))
+
+const AttestantName = styled.div(({ theme }) => ({
+  ...theme.typography.subtitle1,
+  whiteSpace: "nowrap",
+  color: theme.custom.colors.white,
+  lineHeight: "125%",
+}))
+
+const ButtonsContainer = styled.div(({ theme }) => ({
+  display: "flex",
+  justifyContent: "right",
+  margin: "5px auto 0",
+  gap: "16px",
+  [theme.breakpoints.down("md")]: {
+    marginTop: "16px",
+  },
+}))
+
+const InteriorTestimonialCard: React.FC<InternalTestimonialCardProps> = ({
+  resource,
+}) => {
+  return (
+    <QuoteBody>
+      <AttestationBlock>
+        {resource?.quote ??
+          "You have three powerful forces coming together again, the HBCUs, and MIT, Merlot. The need is worldwide for people of all races, underrepresented populations to be able to access information, and to tap in to rich cultural resources to help them in their academic journey."}
+      </AttestationBlock>
+      <AttestantBlock>
+        <AttestantAvatar>
+          <img src={resource.avatar_medium} />
+        </AttestantAvatar>
+        <AttestantNameBlock>
+          <AttestantName>
+            {resource?.attestant_name ?? "Dr. Robbie Melton"}
+          </AttestantName>
+          {resource.title ?? "Some Person"}
+        </AttestantNameBlock>
+      </AttestantBlock>
+    </QuoteBody>
+  )
+}
+
+const TestimonialCard: React.FC<TestimonialCardProps> = ({
+  channels,
+  offerors,
+}) => {
+  const { data } = useTestimonialList({
+    channels: channels,
+    offerors: offerors,
+  })
+  const [slick, setSlick] = React.useState<Slider | null>(null)
+  const responsive = [
+    {
+      breakpoint: theme.breakpoints.values["md"],
+      settings: {
+        adaptiveHeight: true,
+      },
+    },
+  ]
+
+  if (!data) return null
+
+  return (
+    <QuoteContainer>
+      <QuoteBlock>
+        <QuoteLeader>“</QuoteLeader>
+        <Slider
```

Comment:
When there is only 1 testimonial, I don't think we should use a slider. We don't want buttons, and it seems to cause problems (more on this in a sec).

So I suggest: `{data.results.length === 1 ? : < InteriorTestimonialCard /> : <Slider ... />}`


### Carousel misbehaves with a single slide

When there's only one testimonial, I get:

<img width="1272" alt="Screenshot 2024-06-11 at 11 43 35 AM" src="https://github.com/mitodl/mit-open/assets/9010790/47f636e2-2b27-47a0-880a-8108b873d9ad">

Crazy. Works great for 2+ slides.